source/testnet/net/index.rst

```diff
@@ -61,9 +61,9 @@
    desktop-wallet/create-account-desktop
    desktop-wallet/accounts-desktop
    desktop-wallet/create-credentials-file
-   desktop-wallet/shield-gtu-desktop
+   desktop-wallet/shield-ccd-desktop
    desktop-wallet/export-import-desktop
-   desktop-wallet/send-gtu-single-desktop
```

Comment:
Just so you know, this change means the already released links for the sites will change.
We might want to redirect the old link to the new one.